uber/models.py

```diff
@@ -1052,6 +1052,12 @@ class Group(MagModel, TakesPaymentMixin):
     name            = Column(UnicodeText)
     tables          = Column(Numeric, default=0)
     address         = Column(UnicodeText)
```

Comment:
Yeah, the 'international' checkbox is just a way for attendees to bypass filling in the zipcode field. We don't want or need that for dealers.

We... could rename the `address` field to `address1`. Actually do Labs/West even have dealers in the system? If not, we can just drop the column.